cli/package.json

```diff
@@ -37,11 +37,15 @@
     "prompt": "^0.2.14",
     "q": "~1.4.1",
     "recursive-fs": "0.1.4",
+    "rimraf": "^2.5.1",
     "semver": "4.3.6",
     "slash": "1.0.0",
     "update-notifier": "^0.5.0",
     "wordwrap": "1.0.0",
     "yargs": "^3.15.0",
     "yazl": "2.2.2"
+  },
+  "peerDependencies": {
+    "react-native-cli": "^0.1.10"
```

Comment:
This would result in a warning for Cordova users that don't have RN installed? If so, I think we should skip adding it, and instead rely on dynamically requiring the packager at runtime like I mentioned in my other comment.